code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -251,6 +251,17 @@ var/global/list/datum/stack_recipe/cardboard_recipes = list (
 	new/datum/stack_recipe("cardboard cutout", /obj/item/cardboard_cutout, 5),
 )
 
+/obj/item/stack/sheet/cardboard/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/stamp/clown) && !istype(loc, /obj/item/storage))
+		var/atom/droploc = drop_location()
+		if(use(TRUE))
```

Comment:
This should actually be a 1. Because you are using 1 of the stack, not using a TRUE stack - Although they're mathematically the same using a boolean where a number should be confuses future coder.